libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -1,4 +1,4 @@
-<form [formGroup]="searchForm" (submit)="searchBooks()">
+<form [formGroup]="searchForm">
```

Comment:
you don't really need to do this  - having submit here is not going to affect anything . you should allow the user to have both options